docs/lab-examples/lab-examples.md

```diff
@@ -56,6 +56,16 @@ You are ready to deploy!
 containerlab deploy -t <topo-file>
 ```
 
+#### SSH access
+For nodes that come up with `sshd` enabled, the following lines can be added to `~/.ssh/config` file to simplify access and prevent future ssh key warnings:
```

Comment:
can we change `sshd` to just SSH? It feels the details are not that important here in this context.

Also can you please add the the ~/.ssh/config is on the containerlab host system? I bet somebody will try to change it in the context of a node...